code/src/main/java/frc/robot/RobotContainer.java

```diff
@@ -36,8 +40,9 @@
   @Inject
   private RobotContainer(final ILogger logger, IDrivetrainSubsystem drivetrain,
       DrivetrainControllerCommand driveCommand, IIntakeSubsystem intake, DriveArmCommand driveArmCommand,
-      IClimbSubsystem climb, ClimbControllerCommand climbCommand) {
+      IClimbSubsystem climb, ClimbControllerCommand climbCommand, Provider<Easy23StrategyCommand> autonomousCommand) {
```

Comment:
This seems like a problem as our current plan for Glacier peak is to create a time based sequential command that will score 17 points. We are having issues with odometry, and we don't have time to fix it before glacier peak.

Reply:
so this is just a sample, we've not decided on what we'll go with as far as I know. I needed something to test. It doesn't prevent us from coding a different strategy and replacing it.

Reply:
That's why I'm hesitant. I thought we were going to run master at competition, so I would prefer that we don't put it into master.

Reply:
You could add a todo stub and open an issue

Reply:
OK so I'm failing to understand what the issue here is. Its my understanding you have an Auton routine that you want to add. You can simply replace the one I have with the one you want to use. Everything else is unaffected. I want to get this is so I don't have to spend a lot of time with merge conflicts again.

Reply:
Ok, That makes sense